imports/node-app/core-services/payments/queries/getPaymentMethodConfigByName.js

```diff
@@ -0,0 +1,14 @@
+const paymentMethods = {};
```

Comment:
This needs to be `import { paymentMethods } from "../registration.js";` because the function that sets them into this list is in that file.